maven-plugins/enforcer-maven-plugin/src/test/java/io/helidon/build/maven/enforcer/ExcludeTest.java

```diff
@@ -98,4 +100,100 @@ void testContains() {
         excluded = exclude.matches(FileRequest.create("src/resources/sbin/copyright.txt"));
         assertThat("Should not matches directory", excluded, is(false));
     }
+
+    @Test
+    void testNameEnd() {
```

Comment:
Can we have a test or sets of tests dedicated to `.gitignore` ?
I.e. I'd like to see all the use-cases listed at https://git-scm.com/docs/gitignore#_pattern_format

We should also have an integration test that mocks a directory with `.gitignore` file and pseudo versioned files.